apps/site/styles/effects.css

```
@@ -10,3 +10,47 @@ h1.special {
     md:leading-[4rem]
     md:-tracking-[0.06rem];
 }
+
+.turtle {
+  @apply animate-surf
+    absolute
+    z-10
+    translate-x-0
+    translate-y-0;
+}
+
+.turtle img {
+  @apply h-auto
+    w-[200px];
```

Instead of adding media query we can use it directly like this;
```suggestion
    w-24
    md:w-48;
```

Also, instead of using px, I tried to round to the nearest values ​​in the Tailwind's [spacings](https://github.com/tailwindlabs/tailwindcss/blob/main/stubs/config.full.js#L841). We can leave it as px if we don't want it to be scaled